WordPressUI/Ghosts/UIView+Ghost.swift

```diff
@@ -6,6 +6,17 @@ import UIKit
 //
 extension UIView {
 
+    /// Property that defines if a view is ghostable. Defaults set to true.
+    ///
+    public var isGhostable: Bool {
```

Comment:
This property name has got me thinking: 

- Normal use would be to set `isGhostable = false`, since it defaults to `true`
- So you would, normally, add a `isGhostable` entry in IB's KVO interface, and leave it unkarmed

How about if, instead, this is actually called:

- isGhostableDisabled
- skipsGhostable

Thoughts?

Reply:
I've changed to `isGhostableDisabled`. 👍